### DISCUSSION TITLE
Simulations to examine wet water vapor bias in UTLS

### DISCUSSION THREAD

**Discussion by cecilehannay** [NCAR/amwg_dev#308] [Tape recorder simulations]
## Set of simulations to look at wet water vapor bias in Upper Troposhere Lower Stratosphere

We went from a too dry to too wet stratosphere. This bias is discussed in: #300

Here we performed a set of simulations to analyze the impact of tuning F in this bias. 

## Control
- **Link**: #296 

## micro_mg_vtrmi_factor 
- **micro_mg_vtrmi_factor** = 0.5->1.0 
- **Link**: #305

## microp_aero_wsubi_scale 
- **microp_aero_wsubi_scale** = 2.5->1.0 
- **Link**: #306

## micro_mg_dcs 
- **micro_mg_dcs** = 0.00025->0.00050 
- **Link**: #307

## Contacts:
@JulioTBacmeister
@PeterHjortLauritzen
@adamrher

**Comment by islasimpson** [comment]
![h2o](https://github.com/NCAR/amwg_dev/assets/41842751/0ebea805-6050-48e2-b647-b2e093a4a340)

**Comment by justin-richling** [comment]
### micro_mg_vtrmi_factor

[ADF vs control](https://webext.cgd.ucar.edu/FLTHIST/f.cam6_3_112.FLTHIST_v0c.ne30.non-ogw-ubcT-effgw0.3-rdg_beta0.5-vtrmi1.0.001/atm/f.cam6_3_112.FLTHIST_v0c.ne30.non-ogw-ubcT-effgw0.3-rdg_beta0.5-vtrmi1.0.001_1995_2006_vs_f.cam6_3_112.FLTHIST_v0c.ne30.non-ogw-ubcT-effgw0.3-rdg_beta0.5.001_1995_2006/)
[ADF vs obs](https://webext.cgd.ucar.edu/FLTHIST/f.cam6_3_112.FLTHIST_v0c.ne30.non-ogw-ubcT-effgw0.3-rdg_beta0.5-vtrmi1.0.001/atm/f.cam6_3_112.FLTHIST_v0c.ne30.non-ogw-ubcT-effgw0.3-rdg_beta0.5-vtrmi1.0.001_1995_2006_vs_Obs/)

### microp_aero_wsubi_scale

[ADF vs control](https://webext.cgd.ucar.edu/FLTHIST/f.cam6_3_112.FLTHIST_v0c.ne30.non-ogw-ubcT-effgw0.3-rdg_beta0.5-wsubi1.0.001/atm/f.cam6_3_112.FLTHIST_v0c.ne30.non-ogw-ubcT-effgw0.3-rdg_beta0.5-wsubi1.0.001_1995_2006_vs_f.cam6_3_112.FLTHIST_v0c.ne30.non-ogw-ubcT-effgw0.3-rdg_beta0.5.001_1995_2006/)
[ADF vs obs](https://webext.cgd.ucar.edu/FLTHIST/f.cam6_3_112.FLTHIST_v0c.ne30.non-ogw-ubcT-effgw0.3-rdg_beta0.5-wsubi1.0.001/atm/f.cam6_3_112.FLTHIST_v0c.ne30.non-ogw-ubcT-effgw0.3-rdg_beta0.5-wsubi1.0.001_1995_2006_vs_Obs/)

### micro_mg_dcs

[ADF vs control](https://webext.cgd.ucar.edu/FLTHIST/f.cam6_3_112.FLTHIST_v0c.ne30.non-ogw-ubcT-effgw0.3-rdg_beta0.5-dcs500.001/atm/f.cam6_3_112.FLTHIST_v0c.ne30.non-ogw-ubcT-effgw0.3-rdg_beta0.5-dcs500.001_1995_2006_vs_f.cam6_3_112.FLTHIST_v0c.ne30.non-ogw-ubcT-effgw0.3-rdg_beta0.5.001_1995_2006/)
[ADF vs obs](https://webext.cgd.ucar.edu/FLTHIST/f.cam6_3_112.FLTHIST_v0c.ne30.non-ogw-ubcT-effgw0.3-rdg_beta0.5-dcs500.001/atm/f.cam6_3_112.FLTHIST_v0c.ne30.non-ogw-ubcT-effgw0.3-rdg_beta0.5-dcs500.001_1995_2006_vs_Obs/)

**Reply by adamrher** [reply]
Thanks Justin, this is great! 

wsubi has a substantial brightening of clouds in the west pacific/indian ocean. Consider this fact in the context that we need to dim our clouds by ~2 W/m2 in order to get realistic clouds in a B-case. I think this suggests we can consider not increasing wsubi by quite as much (1.0->1.75 instead of 1.0->2.5?), which should also improve the tape recorder.

**Comment by dan800** [comment]
FYI, post the CAM7 meeting this AM, @dkinnison and @tilmes discussed the MT run with interactive chemistry. The word on the street it is still too moist, and so having a self consistent ozone (and thus ozone SW heating) will not solve the problem. @dkinnison will confirm today, but it would be nice if that run could be added to the tape recorder plot. If that is the case we probably should look at other ways to reduce stratospheric water vapor.

**Reply by adamrher** [reply]
Can you provide the cam_tag you're basing this off of? Maybe provide us with the case directory and archive directory so we can understand what this means?

**Reply by adamrher** [reply]
Thanks, I didn't see this run. So you're saying that self-consistent ozone doesn't resolve the moisture bias? OK. But I do wonder if it changed the tape recorder at all, compared to MT non-interactive chemistry that uses L70 O3 forcing.

**Reply by dan800** [reply]
Yes, if @islasimpson can add #310 to the plot above that would be great.

**Comment by tilmes** [comment]
The current MT chemistry run is described here: https://github.com/NCAR/amwg_dev/issues/310

**Reply by dan800** [reply]
Hi Simone, Can you tell me which LT CAM run issue number has the most similar physics settings? I.e. what should I be comparing it to to isolate the difference from running prognostic ozone.

**Reply by dan800** [reply]
Ah, I just checked and it's #288 we should be comparing to, which is not in the comparison, so it's unclear if it improves things or not.

**Reply by tilmes** [reply]
yes #288 is the MT with GHG chemistry

**Reply by islasimpson** [reply]
I had put a comparison with #288 in the other discussion https://github.com/NCAR/amwg_dev/discussions/300.

**Comment by islasimpson** [comment]
#310 is added here...

![h2o](https://github.com/NCAR/amwg_dev/assets/41842751/7b1721dd-1c66-450d-be9e-05239d429480)

**Comment by islasimpson** [comment]
Also a plot examining the influences of hb on water vapor.  Note the different color scale from the plot above...
![hbdiff_impact](https://github.com/NCAR/amwg_dev/assets/41842751/8189b2a4-3aa9-4997-a8ad-b74e3d7c6953)

**Reply by adamrher** [reply]
Thanks Isla! I think we can say that the HB mods don't really make the tape recorder look any worse. It looks like if anything its a bit drier below 30 hPa (above ~30 hPa, it may be a smidge more moist; hard to tell as contours are saturated). All in all, this is welcome news.
